src/encoded/static/components/award.js

```diff
@@ -543,90 +1498,621 @@ ChartRenderer.propTypes = {
     return (
-        <Panel>
-            <PanelHeading>
-                <h4>{award.pi && award.pi.lab ? <span>{award.pi.lab.title}</span> : <span>No PI indicated</span>}</h4>
-                <ProjectBadge award={award} addClasses="badge-heading" />
-            </PanelHeading>
-            <PanelBody>
-                <FetchedData>
-                    <Param name="experiments" url={`/search/?type=Experiment&award.name=${award.name}`} />
-                    <Param name="annotations" url={`/search/?type=Annotation&award.name=${award.name}`} />
-                    <ChartRenderer award={award} />
-                </FetchedData>
-            </PanelBody>
-        </Panel>
+        <SortTablePanel title="Milestones">
+            <SortTable list={award.milestones} columns={milestonesTableColumns} />
+        </SortTablePanel>
     );
 };
 
-AwardCharts.propTypes = {
-    award: PropTypes.object.isRequired, // Award represented by this chart
+MilestonesTable.propTypes = {
+    award: PropTypes.object.isRequired,
 };
 
+// Overall component to render the cumulative line chart
+const ExperimentDate = (props) => {
+    const { experiments, award } = props;
+    let dateReleasedArray = [];
+    let dateSubmittedArray = [];
+    const deduplicatedreleased = {};
+    const deduplicatedsubmitted = {};
+    let label = [];
+    let data = [];
+    let cumulativedatasetReleased = [];
+    let cumulativedatasetSubmitted = [];
+    let accumulatorreleased = 0;
+    let accumulatorsubmitted = 0;
+    let monthreleaseddiff = 0;
+    let monthsubmitteddiff = 0;
+    const fillreleasedDates = [];
+    const fillsubmittedDates = [];
+
+    // Search experiments for month_released and date_submitted in facets
+    if (experiments && experiments.facets && experiments.facets.length) {
+        const monthReleasedFacet = experiments.facets.find(facet => facet.field === 'month_released');
+        const dateSubmittedFacet = experiments.facets.find(facet => facet.field === 'date_submitted');
+        dateReleasedArray = (monthReleasedFacet && monthReleasedFacet.terms && monthReleasedFacet.terms.length) ? monthReleasedFacet.terms : [];
+        dateSubmittedArray = (dateSubmittedFacet && dateSubmittedFacet.terms && dateSubmittedFacet.terms.length) ? dateSubmittedFacet.terms : [];
+    }
+
+    // Use Moment to format arrays of submitted and released date
+    const standardreleasedTerms = dateReleasedArray.map((term) => {
+        const standardDate = moment(term.key, ['MMMM, YYYY', 'YYYY-MM']).format('YYYY-MM');
+        return { key: standardDate, doc_count: term.doc_count };
+    });
 
-const Award = (props) => {
-    const { context } = props;
-    const statuses = [{ status: context.status, title: 'Status' }];
+    const standardsubmittedTerms = dateSubmittedArray.map((term) => {
+        const standardDate = moment(term.key, ['MMMM, YYYY', 'YYYY-MM']).format('YYYY-MM');
+        return { key: standardDate, doc_count: term.doc_count };
+    });
 
```

Comment:
Since these two loops do the same thing, I would implement them just once as an independent function, and pass dateReleasedArray and dateSubmittedArray in two different calls to that function. That function can be nested inside this function to keep the upper variable name scope cleaner, or you can make it its own independent function in the global space if you prefer.